third_party/python/BUILD

```diff
@@ -14,11 +14,7 @@ pip_library(
 
 pip_library(
     name = "protobuf",
-    hashes = [
-        "f958e27563875e4e4374cf4fce3c061f2f7742faa1139acf8def40da07e0ce4e",  # TM Mirror
-        "912fa826e4b9a16947853ca163c18ad530a42a159de357769052443898cae473",  # Public
-        "95dfa3e09ff6a00dd9a02a250d82255d43a2a2b48144c034ee313efd15fd73da",  # GitHub Actions
-    ],
+    hashes = [],
```

Comment:
Can't we remove the `hashes` parameter altogether? It's not compulsory for `pip_library`, and its default value is `None`.